Tools/clinic/clinic.py

```diff
@@ -1717,7 +1717,6 @@ class Block:
     signatures: list[Module | Class | Function] = dc.field(default_factory=list)
     output: Any = None  # TODO: Very dynamic; probably untypeable in its current form?
     indent: str = ''
-    preindent: str = ''
```

Comment:
There's also a section in the Block docstring about *preindent*; we should remove that as well.

Reply:
I reworked the docstring, but didn't remove the section on `preindent` entirely, as the section on `indent` is confusing without it. LMK what you think!